extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -131,19 +132,15 @@
 .i-amphtml-story-tooltip-icon {
   width: 24px !important;
   height: 24px !important;
+  background-size: cover !important;
+  background-position: center !important;
```

Comment:
Could you also add `background-repeat: no-repeat;` for when the icon is super small?